molecule/ccloud/converge.yml

```diff
@@ -11,6 +11,14 @@
       delegate_to: 127.0.0.1
       run_once: true
 
+    - name: Change kafka_controller Group Name in Inventory File
+      lineinfile:
+        path: "{{inventory_dir}}/ansible_inventory.yml"
+        regexp: 'kafka_controller:'
+        line: 'kafka_controller_ccloud:'
+      delegate_to: 127.0.0.1
```

Comment:
This should get executed only if kafka_controller is in the host group. OR may be some better way to identify if this is kraf cluster.

Reply:
It's actually looking in inventory file and finding `kafka_controller`. That's similar to using something like this->  `when: {{ 'kafka_controller' in groups.keys() and groups['kafka_controller']|length >0 }}`
And if the entry is not there, it will not do anything.